custom_components/bosch_alarm/const.py

```diff
@@ -2,5 +2,6 @@
 
 DOMAIN = 'bosch_alarm'
 HISTORY_ATTR = 'history'
+DATETIME_ATTR = "datetime"
```

Comment:
Single quotes, please

Also, if it's only used in alarm_control_panel.py, we should probably just put it there -- there's already some precedent with FAULTED_POINTS_ATTR